apps/script/HoC2023AiGenerateWeights.py

```diff
@@ -161,3 +184,85 @@
     
 with open("apps/static/dance/ai/model/cached-spacy-foreground-map.json", "w") as json_file:
     json_file.write(json.dumps(foreground_output))
+
+# Below are background effects, palettes and foreground effects in the order they appear in the dropdowns.
+# When this file is run and the three cached data files are replaced in apps/static/dance/ai/model,
+# please update comments lists below with output of this script.
+# This script is run from the code-dot-org root directory.
+# python apps/script/HoC2023AiGenerateWeights.py
+
+# BACKGROUND EFFECTS
+# python_name | blockly_id | blockly_user_facing_name
```

Comment:
Thoughts on something like "descriptive_name"? Or, if we want to be verbose and sort of similar to the blockly one like `[technology]_[usage]_name` , `python_descriptive_name`? `python_name` by itself seems like it could be confusing to future readers. But I can get on board if you think that's our best option

Reply:
Sure - I think @sanchitmalhotra126 had a similar suggestions. Maybe I can use `ai_model_name`?

Reply:
My concern about that is it kind of sounds like "this is the name of the model", although it might make sense in context. `model_descriptive_name`? Up to you :)

Reply:
I like `model_descriptive_name` - thanks so much!